Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityController.cs

```diff
@@ -516,7 +529,118 @@ private void UpdateMenuData(InteractionSourceState interactionSourceState, Mixed
             }
         }
 
-        #endregion Update data functions
+        /// <summary>
+        /// Update the Controller Model
+        /// </summary>
+        /// <param name="interactionSourceState"></param>
+        private void UpdateControllerModel(InteractionSourceState interactionSourceState)
+        {
+            if (controllerModelInitialized ||
+                !(MixedRealityToolkit.Instance?.ActiveProfile?.InputSystemProfile?.ControllerVisualizationProfile?.GetUseDefaultModelsOverride(GetType(), ControllerHandedness) ?? true))
+            {
+                controllerModelInitialized = true;
+                return;
+            }
+
+            controllerModelInitialized = true;
+            CreateControllerModelFromPlatformSDK(interactionSourceState.source.id);
+        }
+
+#endregion Update data functions
+
+#region Controller model functions
+
+        protected override void TryRenderControllerModel(Type controllerType)
```

Comment:
Try methods should have a bool return value, correct?

Reply:
on board with changing this behavior. I think i originally changed from private to protected but am on board with updating behavior to be more clear

Reply:
Ideally I'd like to see the changes to the input system that were required (see my fork) before getting change like this in.

Reply:
Hm, the input system changes definitely look good (it seems like they will allow controller animations to work). But i think that may be beyond the scope of getting the controller to render. I talked some with Kurtis and we're not sure if controller animations meet the RC1 requirements. It may be enough to first get the controller appearing. Then we can fix/polish having the animations working.

What amount of churn occurred in the input system? I would guess that merging your changes in with other work around articulated hands that will be hitting the public repo could be non-trivial. @luis-valverde-ms is the best contact for helping with an input system change/integration story

Reply:
Good question. Most of those changes involved moving the controller data providers into the a registry provided by the service. [Hard to say now that the fork has diverged so much](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3252). Part of why I was stressing getting [those changes](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3116) addressed months ago.

Reply:
I think this is beyond the scope of this review. Input changes should be discussed with @luis-valverde-ms . I don't know that I will own input system design/am the best contact for these design discussions. I am mainly responsible for enabling this gltf work.

Reply:
@StephenHodgson, I like very much the idea of having a separate profile for controller data providers but I'm reluctant to take breaking changes of that extent this close to RC1. We should definitely keep it in sight for the v3.

Going off topic here but would you mind, if you have the time, giving me a quick summary of all other input changes in your branch? I want to keep them in mind for v3 too.

Reply:
@luis-valverde-ms see #3116

Reply:
@StephenHodgson, I was wondering if there were other input related changes in your fork, besides that one, that you thought interesting bringing over.